docker_compose_swarm_mode.py

```diff
@@ -139,8 +139,12 @@ def container_name():
                 def hostname():
                     pass  # unsupported; waiting for https://github.com/docker/docker/issues/24877
 
+                def labels():
+                    for label in value:
```

Comment:
According to the [docs](https://docs.docker.com/compose/compose-file/#/labels), `labels` key can have either list or dict as a value. We should check if it's a dict and convert it to the list in this case.

P.S.: Please don't mix different features in a single commit so that they can be merged separately.